### PR TITLE
Sphinx autodocs setup

### PR DESCRIPTION
**Pull request by MaxHellstrom**
Completed sphinx autodoc setup. The interesting stuff is in commit 46516e87925a4b0705b234ff85314c2d6fd65401.

### CODE REVIEW

File: docs/source/user/install.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 (install/installation)=
-# Installing PySkinDose
+# installation
```
Comment (BwKodex) [inline docs/source/user/install.md:2]: Shouldn't this be capitalized?

File: docs/source/user/get_started.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+getting started
```
Comment (BwKodex) [inline docs/source/user/get_started.rst:1]: Shouldn't this be capitalized?

File: docs/source/user/description.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+what is PySkinDose?
```
Comment (BwKodex) [inline docs/source/user/description.rst:1]: Shouldn't this be capitalized?

File: docs/source/user/contribute.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+contributions and collaborations
```
Comment (BwKodex) [inline docs/source/user/contribute.rst:1]: Shouldn't this be capitalized?

File: src/pyskindose/settings_pyskindose.py [hunk 5]
```diff
@@ -27,7 +27,7 @@ class PyskindoseSettings:
         presents the result in a skindose map.
 
         mode = "plot_setup" plots the geometry (patient, table, pad and beam
-        in starting position, i.e., before any RDSR data has been added.) This
+        in starting position, i.e., before awny RDSR data has been added.) This
```
Comment (BwKodex) [inline src/pyskindose/settings_pyskindose.py:30]: Check spelling

File: src/table_translation.json [hunk 6]
```diff
@@ -0,0 +1 @@
..."9":0,"10":0,"11":0,"12":0},"FS_lat":{"0":20,"1":20,"2":20,"3":20,"4":20,"5":20,"6":20,"7":20,"8":20,"9":20,"10":20,"11":20,"12":20},"FS_long":{"0":20,"1":20,"2":20,"3":20,"4":20,"5":20,"6":20,"7":20,"8":20,"9":20,"10":20,"11":20,"12":20},"DSD":{"0":107,"1":107,"2":107,"3":107,"4":107,"5":107,"6":107,"7":107,"8":107,"9":107,"10":107,"11":107,"12":107},"DSI":{"0":78.5,"1":78.5,"2":78.5,"3":78.5,"4":78.5,"5":78.5,"6":78.5,"7":78.5,"8":78.5,"9":78.5,"10":78.5,"11":78.5,"12":78.5},"DID":{"0":28.5,"1":28.5,"2":28.5,"3":28.5,"4":28.5,"5":28.5,"6":28.5,"7":28.5,"8":28.5,"9":28.5,"10":28.5,"11":28.5,"12":28.5},"DSIRP":{"0":63.5,"1":63.5,"2":63.5,"3":63.5,"4":63.5,"5":63.5,"6":63.5,"7":63.5,"8":63.5,"9":63.5,"10":63.5,"11":63.5,"12":63.5},"K_IRP":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"DSL":{"0":40,"1":40,"2":40,"3":40,"4":40,"5":40,"6":40,"7":40,"8":40,"9":40,"10":40,"11":40,"12":40},"Ap1":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"Ap2":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"Ap3":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0}}
```
Comment (BwKodex) [inline src/table_translation.json:1]: Should we make this human-readable/prettify it?

**Review by BwKodex — COMMENTED** [review]
Nicely done. Just a few nitpicky comments.

File: docs/source/user/install.md (hunk 1 above)
Comment (MaxHellstrom) [reply docs/source/user/install.md:2]: resolved

File: docs/source/user/get_started.rst (hunk 2 above)
Comment (MaxHellstrom) [reply docs/source/user/get_started.rst:1]: resolved

File: docs/source/user/contribute.rst (hunk 4 above)
Comment (MaxHellstrom) [reply docs/source/user/contribute.rst:1]: resolved

File: src/table_translation.json (hunk 6 above)
Comment (MaxHellstrom) [reply src/table_translation.json:1]: resolved

**Comment by MaxHellstrom** [comment]
all comments resolved

**Review by BwKodex — APPROVED** [review]

**Pull request merged**